pkg/agent/agent_windows.go

```diff
@@ -307,3 +446,26 @@ func (i *Initializer) setInterfaceMTU(iface string, mtu int) error {
 	}
 	return util.SetInterfaceMTU(iface, mtu)
 }
+
+// installVMFlows configures default flows between uplink and host port,
+// so that antrea-agent can connect to antrea-controller.
+func (i *Initializer) installVMFlows() error {
+	var uplinkOFPort int32
+	var hostOFPort int32
+	var hostIfName string
+	hostIfConfig, found := i.ifaceStore.GetInterfaceByName(i.nodeConfig.UplinkNetConfig.Name)
+	if found {
+		hostIfName = hostIfConfig.InterfaceName
+		hostOFPort = hostIfConfig.OVSPortConfig.OFPort
+		uplinkOFPort = hostIfConfig.EntityInterfaceConfig.UplinkPort.OFPort
+		klog.InfoS("Installing flows", "hostIfName", hostIfName, "hostOFPort", hostOFPort, "uplinkOFPort", uplinkOFPort)
+		if err := i.ofClient.InstallVMUplinkFlows(hostIfName, hostOFPort, uplinkOFPort); err != nil {
+			err := fmt.Errorf("failed to install host fows for interface %s", hostIfName)
```

Comment:
return fmt.Errorf("failed to install host fows for interface %s", hostIfName)

Reply:
Done